examples/deck.gl/3d-tiles/app.js

```diff
@@ -16,7 +16,6 @@ import {DracoLoader} from '@loaders.gl/draco';
 import {GLTFLoader} from '@loaders.gl/gltf';
 
 import ControlPanel from './components/control-panel';
-import fileDrop from './components/file-drop';
```

Comment:
Remove the file too, and any mentions of `droppedFile` field in e.g. control panel?